src/components/scenegraph/MotionCapture.js

```diff
@@ -165,7 +165,7 @@ export default class MotionCapture extends React.Component {
     document.addEventListener('keyup', function keyboardStart () {
       textEntity.parentNode.removeChild(textEntity);
       self.countdownRecording();
-      sceneEl.removeEventListener('keyup', buttonStart);
+      sceneEl.removeEventListener('keyup', keyboardStart);
```

Comment:
`sceneEl` => `document` as you're adding the listener to `document` isn't it?

Reply:
oops, fixed again